headers/bytecode_api.h

```diff
@@ -172,7 +172,10 @@ enum FunctionalityLevels {
 
     FUNC_LEVEL_1_2 = 190, /**< LibClamAV release 1.2.0 */
 
-    FUNC_LEVEL_1_3 = 200, /**< LibClamAV release 1.3.0 */
+    FUNC_LEVEL_1_3 = 200,   /**< LibClamAV release 1.3.0 */
+    FUNC_LEVEL_1_3_1 = 201, /**< LibClamAV release 1.3.1 */
+
+    FUNC_LEVEL_1_4 = 210, /**< LibClamAV release 1.4.0 */
```

Comment:
There are some missing patch versions for 0.103, 1.0, 1.1, 1.2

https://github.com/Cisco-Talos/clamav/blob/main/libclamav/bytecode_api.h#L157-L192